tekton_kfp_guide.md

```diff
@@ -11,24 +14,27 @@
   * [App layout](#app-layout)
 - [Troubleshooting](#troubleshooting)
 
-## Prequisites
+## Installation Targets and Prequisites
 
 A Kubernetes cluster `v1.16` that has least 8 vCPU and 16 GB memory. 
 
-1. Using *IBM Cloud Kubernetes Service (IKS)*:
+### IBM Cloud Kubernetes Service (IKS):
```

Comment:
Because of the Travis syntax test, if we are going to turn this into header, then we need to remove the colons.
```suggestion
### IBM Cloud Kubernetes Service (IKS)
```